code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm

```diff
@@ -305,7 +305,7 @@
 /obj/structure/machinery/cm_vending/sorted/cargo_guns/squad/populate_product_list(scale)
 	listed_products = list(
 		list("FOOD", -1, null, null),
-		list("MRE", round(scale * 5), /obj/item/storage/box/MRE, VENDOR_ITEM_REGULAR),
+		list("MRE", round(scale * 5), /obj/item/storage/box/MRE/nofluff, VENDOR_ITEM_REGULAR),
 		list("MRE Box", round(scale * 1), /obj/item/ammo_box/magazine/misc/mre, VENDOR_ITEM_REGULAR),
```

Comment:
isn't this MRE Box suppose to be no fluff to?
/obj/item/storage/box/MRE/nofluff

Reply:
the box contains the no-fluff MREs

Reply:
naming in the vendor was  wrong? it's "MRE" when it's should be "MRE Box" and bellow it should be the opposite?

Reply:
MRE box is the big box with all the items. So its technically correct as far as the average player is concerned

Reply:
yea your right after looking into it it's just the path for the object that made me very confused...
sorry